dhis2/src/main/java/org/motechproject/dhis2/rest/service/impl/DhisWebServiceImpl.java

```diff
@@ -411,7 +413,13 @@ private DhisStatusResponse createEntity(Settings settings, String uri, String js
         DhisStatusResponse status;
 
         try (InputStream content = getContentForResponse(response)) {
-            status = new ObjectMapper().readValue(content, DhisStatusResponse.class);
+            String contentString = IOUtils.toString(content);
+            status = new ObjectMapper().readValue(contentString, DhisStatusResponse.class);
+            if (status.getStatus() == DhisStatus.ERROR) {
+                String msg = String.format("DHIS2 status response error details: %s", contentString);
+                statusMessageService.warn(msg, MODULE_NAME);
```

Comment:
I have read once again ticket description and I think that we should omit logging this, just throw exception and task will fail